cmake/developer_package/frontends/frontends.cmake

```diff
@@ -202,7 +202,8 @@ macro(ov_add_frontend)
         # frontend's CMakeLists.txt must define its own custom 'ov_ncc_naming_style' step
     else()
         ov_ncc_naming_style(FOR_TARGET ${TARGET_NAME}
-                            SOURCE_DIRECTORY "${frontend_root_dir}/include"
+                            SOURCE_DIRECTORIES "${frontend_root_dir}/include"
+                                            #    "${frontend_root_dir}/src"
```

Comment:
@andrei-kochin you need to uncomment this line to check naming style for ALL frontends (except ONNX, because it has custom rule, i.e. `ov_add_frontend` is called with `SKIP_NCC_STYLE` option)